src/test/java/com/ticketservice/dao/TicketServiceTest.java

```diff
@@ -0,0 +1,118 @@
+package com.ticketservice.dao;
+
+import com.sun.tools.javac.util.List;
+import com.ticketservice.dao.model.Ticket;
+import com.ticketservice.dao.repository.TicketRepository;
+import com.ticketservice.dao.services.TicketService;
+import jakarta.persistence.EntityNotFoundException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.dao.DataIntegrityViolationException;
+
+
+import java.util.Optional;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class TicketServiceTest {
+
+    private static TicketRepository ticketRepository;
+    private static TicketService ticketService;
+
+    @BeforeAll
+    public static void init() {
+        ticketRepository = mock(TicketRepository.class);
+        ticketService = new TicketService(ticketRepository);
+    }
+
+
+    @Test
+    public void saveTicket_SavedExpected() {
+        Ticket ticketToBeSaved = new Ticket();
+
+        Mockito.when(ticketRepository.save(ticketToBeSaved)).thenReturn(ticketToBeSaved);
```

Comment:
Since you already had static import for Mockinto.when you should use it instead of specify full name